new-integration-tests/src/test/resources/istio/istio-cdt-http-template-service.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-cdt-http-gateway
+  namespace: NAMESPACE
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - hosts:
+        - '*'
+      port:
+        name: http
+        number: 80
+        protocol: HTTP
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: istio-cdt-http-virtualservice
+  namespace: NAMESPACE
+spec:
+  gateways:
+    - istio-cdt-http-gateway
+  hosts:
+    - 'domain1-NAMESPACE.org'
+  http:
+    - match:
+        - uri:
+            prefix: /console
+        - uri:
```

Comment:
Do you need this uri prefix ?

Reply:
you mean the management prefix. I guess not. Will remove it.